### ISSUE TITLE
Fix CreateSampleWorkspace

### ISSUE THREAD

**Issue by OwenArnold** [mantidproject/mantid#11966]
This issue was originally [TRAC 11127](http://trac.mantidproject.org/mantid/ticket/11127)

I've just had to fix CreateSampleWorkspace because the reference frame does not correspond with the acutal layout of the test instrument made.

I now find that we have a property called "BankPixelWidth" on this algorithm. This doesn't actually control the pixel width. It controls the number of pixels! See the source code.

The usage examples are good. But It would be extremely helpful to see what the layout of the instrument really is. If you create a sample workspace around an instrument it is often useful to be able to calculate what the L1 & L2 distances are to support your calculations. There is no way of finding that out at present aside from inspecting the source code. I suggest having a schematic (including source, sample locations and detectors etc) so that people can see what sort of instrument and workspace they're getting back. 

In summary:

* Fix the pixel size issue one way or the other, but allow users to calculate size from number (vica versa) by including the Bank Width height somewhere, probably easiest in the documentation.
* Add a schematic of the instrument layout to the documentation

http://docs.mantidproject.org/nightly/algorithms/CreateSampleWorkspace-v1.html
